build/templates/_converters.py.mako

```diff
@@ -153,6 +153,59 @@ def convert_repeated_capabilities_without_prefix(repeated_capability):
     return ','.join(convert_repeated_capabilities(repeated_capability, ''))
 
 
+def convert_single_group_repeated_capabilities(repeated_capability):
+    '''Convert a repeated capabilities string, with at most one prefix (that ends with '/') and without any comma, to a list
+
+    Args:
+        repeated_capability (str) - Supported types (with an optional prefix):
+            - range (using '-' or ':')
+            - single item
+
+    Returns:
+        rep_cap (str) - comma delimited list of strings of the expanded repeated capability items
+    '''
+    if '/' in repeated_capability:
+        split_index = repeated_capability.find('/') + 1
+        prefix = repeated_capability[:split_index]
+        return convert_repeated_capabilities(repeated_capability[split_index:], prefix)
+    return convert_repeated_capabilities(repeated_capability)
+
+
+def convert_channels_repeated_capabilities(
+    channels_repeated_capability,
+    first_channel_name
+):
+    '''Convert a channels repeated capabilities string, possibly with no or multiple prefixes (each ends with '/'), to a list
```

Comment:
maybe add example to docstring

Reply:
I have added examples to the docstring of `convert_channels_repeated_capabilities()` in my latest commit.